recipe/meta.yaml

```diff
@@ -40,6 +40,7 @@ test:
     - if not exist %PREFIX%\Library\bin\libmatio.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\libmatio.lib exit 1  # [win]
     - if not exist %PREFIX%\Library\include\matio.h exit 1  # [win]
+    - set xx=lib /list libmatio.lib # [win] 
```

Comment:
The linter is requesting another space before `# [win]`. Would get rid of any trailing whitespace as well.